wrapped-bridged-usdc/src/lib.rs

```diff
@@ -0,0 +1,120 @@
+#![no_std]
+
+elrond_wasm::imports!();
+
+#[elrond_wasm::contract]
+pub trait WrappedBridgedUsdc {
+    #[init]
+    fn init(&self, universal_bridged_usdc_token_id: TokenIdentifier) {
+        self.universal_bridged_usdc_token_id()
```

Comment:
I know that now we have this set. The ownership over the tokenID must be given for this contract.